content/how-to/cloud-servers/multifactor-authentication-from-the-cloud-control-panel/index.md

```diff
@@ -89,37 +89,42 @@ To register and verify an SMS device, use the following steps:
 
 2. In the upper-right corner of the Cloud Control Panel, click the **user** menu
    and select **My Profile & Settings**.
+   
+   <img width="383" alt="chrome_kE2DgyCefB" src="https://user-images.githubusercontent.com/96761530/160662203-c5eabb2d-29f5-4d72-aaa0-9109016ffb21.png">
 
 3. In the **Security Settings** section, select **Enable** for **Multi-factor
    authentication**.
+   
+   <img width="467" alt="chrome_bz6GjpL2wM" src="https://user-images.githubusercontent.com/96761530/160662363-de2b77ea-7a81-434a-9a01-b05a7f3583c0.png">
```

Comment:
I think this is the image showing the prod design test account number. It would probably be safer if we don't show that account number in this image. Perhaps you can crop it out.

Reply:
Alright

Reply:
Updated...